manim/scene/scene_file_writer.py

```diff
@@ -567,37 +559,46 @@ def combine_files(
             for pf_path in input_files:
                 pf_path = Path(pf_path).as_posix()
                 fp.write(f"file 'file:{pf_path}'\n")
-        commands = [
-            config.ffmpeg_executable,
-            "-y",  # overwrite output file if it exists
-            "-f",
-            "concat",
-            "-safe",
-            "0",
-            "-i",
-            str(file_list),
-            "-loglevel",
-            config.ffmpeg_loglevel.lower(),
-            "-metadata",
-            f"comment=Rendered with Manim Community v{__version__}",
-            "-nostdin",
-        ]
 
+        av_options = {
+            "safe": "0",
+            "metadata": f"comment=Rendered with Manim Community v{__version__}",
+        }
         if create_gif:
-            commands += [
-                "-vf",
-                f"fps={np.clip(config['frame_rate'], 1, 50)},split[s0][s1];[s0]palettegen=stats_mode=diff[p];[s1][p]paletteuse=dither=bayer:bayer_scale=5:diff_mode=rectangle",
-            ]
-        else:
-            commands += ["-c", "copy"]
+            av_options["vf"] = (  # add video filter (is there a better way to do this?)
+                f"fps={np.clip(config['frame_rate'], 1, 50)},"
+                "split[s0][s1];[s0]palettegen=stats_mode=diff[p];"
+                "[s1][p]paletteuse=dither=bayer:bayer_scale=5:diff_mode=rectangle"
+            )
```

Comment:
Seems like video filters must be applied separately: https://github.com/PyAV-Org/PyAV/issues/239

Reply:
Yes, I've tried the `Graph`-construction that you have to push/pull frames through initially, but was happy when you came up with the alternative `options`-approach.

Well, seems like we'll have to look into it regardless.

Reply:
But does the `options` approach work? I'm trying to move Manim Slides to use `av` too, and cannot make the reverse filter work.